test/e2e/utils/chaise.page.js

```diff
@@ -873,6 +873,25 @@ function chaisePage() {
     this.waitForElementCondition = function(condition, timeout) {
         return browser.wait(condition, timeout || browser.params.defaultTimeout);
     };
+
+    this.performLogin = function(cookie, defer) {
+
+        defer = defer || require('q').defer();
+
+        browser.get(process.env.CHAISE_BASE_URL + "/login/");
+        browser.ignoreSynchronization = true;
+          
+        browser.wait(protractor.ExpectedConditions.visibilityOf(element(by.id("loginApp"))), browser.params.defaultTimeout).then(function() {
+            browser.driver.executeScript('document.cookie="' + cookie + ';path=/;' + (process.env.TRAVIS ? '"' : 'secure;"')).then(function() {
+              browser.ignoreSynchronization = false;
+              defer.resolve();
+            });
+        }, function() {
+            defer.reject();
```

Comment:
As mentioned in the e2e test wiki regarding [syncronization issues](https://github.com/informatics-isi-edu/chaise/wiki/E2E-tests-guide#synchronization-issues) we don't use Angular Router so thats the reason we set it to true. 

Once we have visited the page and set the document cookie successfully, I set it to false on L886. This is so that we don't alter the synchronization state and bring it back to the state it was before.